### ISSUE TITLE
Publicize feature development plans for future openhands versions

### ISSUE THREAD

**Issue by ryx2** [All-Hands-AI/OpenHands#5191]
**What problem or use case are you trying to solve?**

Is there a way to make public what the next features to be developed by openhands are? The responses I got when I asked Qs about the enterprise plan had a lot of information about the future of openhand's direction, so I know these plans exist, but maybe it'd help open source contributors to be able to focus their development energy on things aligned with the openhands direction
For an example, currently, we are on release 0.14.2, there could be a section in the main README.md that says:

**Describe the UX of the solution you'd like**

```
Upcoming features

[0.15]
Feature 1 [link to issue A,B,C][link to PR X,Y,Z]
Feature 2 [link to issue D,E][link to PR X2, Y2]

[0.16]
Feature 3 [link to issue F]
```

This way, open source contributors like myself could try and cater PRs towards things that the open hands team is currently focused on?

**Do you have thoughts on the technical implementation?**

**Describe alternatives you've considered**

**Additional context**

**Comment by rbren** [comment]
@ryx2 we have a rough roadmap here: https://github.com/orgs/All-Hands-AI/projects/1

We'll be discussing this in the maintainers meeting today at 2pm ET if you'd like to join! meet.google.com/xvv-bdgs-snt

**Comment by neubig** [comment]
Fixed by #5192! Thanks for the contribution.
